core/src/services/huggingface/mod.rs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+mod backend;
+pub use backend::HuggingFaceBuilder as HuggingFace;
```

Comment:
If we want to make it name after `HuggingFace`, we should use `hugging_face` as it scheme.

Please make sure they are consistent:

- `HuggingFace`, `hugging_face`, `hugging-face`
- `Huggingface`, `huggingface`

I prefer to use the second one.

Reply:
I would also advocate for `huggingface` if possible :pray: